Assets/BML/ScriptableObjectCore/Scripts/Variables/LootTableVariable.cs

```diff
@@ -21,7 +21,9 @@ public enum LootTableKey
         CommonOre2 = 1 << 4,
         RareOre = 1 << 5,
         Bomb = 1 << 6,
-        HealthTemp = 1 << 7,
+        HealthTemp2 = 1 << 7,
+        HealthTemp = 1 << 8,
```

Comment:
Why is HealthTemp2 before HealthTemp? lol

Reply:
1. These keys suck
2. I assumed Health2 referred to a full heart, so I made everything that dropped a full temp heart, HealthTemp2 which was previously HealthTemp